frontend/layouts/shelter/ShelterLoginLayout/index.tsx

```diff
@@ -4,7 +4,11 @@ import styles from "./ShelterLoginLayout.module.css";
 const { Header, Content, Footer } = Layout;
 const logo = "/logo.png";
 
-const ShelterHomeLayout = ({ children }) => {
+type ShelterHomeLayoutProps = {
```

Comment:
type is given long name as the file contains multiple JSX functions. i.e. ShelterHomeLayoutProps vs Props